testing/gcin/APKBUILD

```diff
@@ -0,0 +1,64 @@
+# Contributor: Felix Yan <felixonmars@archlinux.org>
+# Contributor: Gaetan Bisson <bisson@archlinux.org>
+# Contributor: Andreas Radke <andyrtr@archlinux.org>
+# Contributor: damir <damir@archlinux.org>
+# above are from Arch Linux
```

Comment:
if it is based on their pkgbuild, it may be reasonable to keep the contributor lines.